packages/flutter/lib/src/material/text_field.dart

```diff
@@ -348,6 +348,7 @@ class TextField extends StatefulWidget {
     this.autofocus = false,
     this.obscuringCharacter = '•',
     this.obscureText = false,
+    this.forceCloseConnectionOnBlur = false,
```

Comment:
I agree, connection should be an implementation detail. I changed this to `forceSubmitOnFocusLost`, because the onSubmitted callback will be invoked when the user clicks out of the text field.